debian/snapd.install

```diff
@@ -1,4 +1,5 @@
 /usr/bin/snap
+/usr/bin/snapctl
```

Comment:
We probably at least want it in PATH for the hook being executed, since that's where we anticipate it being called (at least for now-- not sure what the long-term vision for this tool is). We could still put it in `/usr/lib/snapd/` but then getting it the path would require `snap run` to add that to the environment. What do you think? @niemeyer do you have any thoughts?